evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -367,6 +412,28 @@ contract Operator is
     assert(linkToken.transfer(msg.sender, payment));
   }
 
+  // PUBLIC FUNCTIONS
+
+  /**
+   * @notice Create a new forwarder contract
+   * @dev This function uses create2 to deploy at a predetermined address.
+   * @return addr deployed address
+   */
+  function createForwarder()
+    public
+    onlyAuthorizedSender()
+    returns (
+      address addr
+    )
+  {
+    bytes32 salt = bytes32(s_forwardersList.length);
+    OperatorForwarder forwarder = new OperatorForwarder{salt: salt}(address(linkToken));
```

Comment:
Doesn't look like we're using the recently added `OperatorForwarderDeployer.sol` yet. Let's swap in a delegate call to that here?

Reply:
I don't think we want to use that here.

`OperatorForwarderDeployer.sol` is a test contract created in this PR. It acts as a dummy `Operator.sol` so that the `OperatorForwarder` can be deployed for testing because it calls to `OperatorInterface(msg.sender).getAuthorizedSenders()` in its constructor.